src/astro/sql/operators/sql_decorator.py

```diff
@@ -166,6 +166,10 @@ def handle_schema(self, output_table_name):
         """
         if self.conn_type == "postgres" and self.schema:
             output_table_name = self.schema + "." + output_table_name
+        elif self.conn_type == "snowflake" and self.schema and "." not in self.sql:
```

Comment:
Why are we checking based on  - "." not in self.sql?

Reply:
Because if the user has already identified the schema and database in the table_name then we don't need to add it.

see here: https://github.com/astro-projects/astro/pull/68#discussion_r799533367